conditions/CephClusterWarningState.json

```diff
@@ -0,0 +1,17 @@
+{
+  "conditions": [
+    {
+      "type": "alert_is_firing",
+      "alert": {
+        "name": "CephClusterWarningState"
+      }
+    }
+  ],
+  "gathering_functions": {
+    "logs_of_namespace": {
```

Comment:
I guess this should be `containers_logs` right.

Reply:
Not really. I mean, we could, but to adapt the `containers_logs` to get the logs from a non-alert-triggering would be much more complicated. Currently, the `containers_log` only gets logs from the pod that is triggering the alert.

Reply:
OK I completely forgot about the `logs_of_namespace` so you can ignore my comment :) Sorry

Reply:
So how many files does this introduce in our archive please?

Reply:
3 files, since the default deployment creates 3 `rook-ceph-osd-*` with `osd` container.